ownCloud/Client/Viewer/PreviewViewController.swift

```diff
@@ -0,0 +1,139 @@
+//
+//  PreviewViewController.swift
+//  ownCloud
+//
+//  Created by Michael Neuwert on 27.08.2019.
+//  Copyright © 2019 ownCloud GmbH. All rights reserved.
+//
+
+/*
+* Copyright (C) 2018, ownCloud GmbH.
+*
+* This code is covered by the GNU Public License Version 3.
+*
+* For distribution utilizing Apple mechanisms please see https://owncloud.org/contribute/iOS-license-exception/
+* You should have received a copy of this license along with this program. If not, see <http://www.gnu.org/licenses/gpl-3.0.en.html>.
+*
+*/
+
+import UIKit
+import ownCloudSDK
+import QuickLook
+
+class PreviewViewController : DisplayViewController, QLPreviewControllerDataSource, QLPreviewControllerDelegate {
+
+	private var qlPreviewController: QLPreviewController?
+	var tapToHideBarsGestureRecognizer: UITapGestureRecognizer!
```

Comment:
Since this gesture recognizer is used for both hiding and showing the bars, maybe rename this to `showHideTapGestureRecognizer` (or similar) to make the name more descriptive.

Reply:
@felix-schwarz done